### PR TITLE
[image_picker] Add web support to the example app.

### PR DESCRIPTION
**Pull request by ditman**
## Description

This plugin adds web support to the `example` app, following [the instructions](https://pub.dev/packages/image_picker_for_web) of the new package.

The web-app corresponding to this PR has been deployed here:

https://dit-picker-tests.web.app

## Related Issues

* https://github.com/flutter/plugins/pull/2767

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/image_picker/image_picker/example/lib/main.dart [hunk 1]
```diff
@@ -139,7 +143,11 @@ class _MyHomePageState extends State<MyHomePage> {
       return retrieveError;
     }
     if (_imageFile != null) {
-      return Image.file(File(_imageFile.path));
+      if (kIsWeb) {
+        return Image.network(_imageFile.path);
```
Comment (cyanglaz) [inline packages/image_picker/image_picker/example/lib/main.dart:147]: nits: Add a comment explaining why we need to use .network for web.

Comment (ditman) [reply packages/image_picker/image_picker/example/lib/main.dart:147]: I'm going to point people to the README of the plugin, where this is already explained:

https://pub.dev/packages/image_picker#getting-ready-for-the-web-platform

Comment (ditman) [reply packages/image_picker/image_picker/example/lib/main.dart:149]: Done

**Comment by ditman** [comment]
Thanks for the super fast review @cyanglaz, I'll merge this as soon as it goes green!

**Pull request merged**